### ISSUE TITLE
[Bug]: Crash when loading world on latest version

### ISSUE THREAD

**Issue by kyleyee52** [Zepalesque/Aether-Redux-Issue-Tracker#122]
### Minecraft Version

1.20.1

### Mod Version

2.0.16

### Forge/Neoforge Version

47.3.1

### What happened?

Attempting to load a singleplayer world results in a CTD.

### What was the expected result?

World loads normally and I can play the game.

### Steps to Reproduce

Load up a singleplayer world. Either existing or creating a new one.

### Log Output (if applicable)

[latest.log](https://github.com/user-attachments/files/15750037/latest.log)

### Crash Report (if applicable)

[crash-2024-06-08_20.54.52-client.txt](https://github.com/user-attachments/files/15750039/crash-2024-06-08_20.54.52-client.txt)

### Is this a compatibility issue with another mod?

Supplementaries

### Additional Context/Information

Seems to be a compatibility issue with Supplementaries. The world loads without Supplementaries and loads with Supplementaries if the previous version (2.0.15) of The Aether Redux is installed.

### Make sure to confirm these as well before submitting the report:

- [X] The issue persists on the latest version of the mod.
- [X] All version numbers are properly recorded, no instances of ‘latest’
- [X] All details are reported clearly and concisely
- [X] This issue is not a duplicate of another issue
- [X] If the issue is a modpack-related issue, it only occurs *with* Redux installed, and not without
- [X] If the issue is a modpack-related issue, it has been reported to the modpack devs as well

**Comment by Groupix05** [comment]
same here

**Comment by Zepalesque** [comment]
ahhh heck yep I'll fix this in 2.0.16b

**Comment by evanbones** [comment]
Confirming this issue for me too, thanks for continuing to develop the mod!

**Comment by dimensionmaster20** [comment]
same issue here. according to my own logs it seems that aether redux is loading the sheets class too early which messes with supplementaries and potentially other mods.

**Comment by Zepalesque** [comment]
yeah I saw that, I have a fix in mind but I’ll have to test to make sure it works properly with supplementaries itself, plus waiting on a response from another person about another bug

**Comment by Zepalesque** [comment]
fixed, as of 2.0.16b

**Comment by Darkmega18** [comment]
very gooood. this was stopping me from playing yesterday.
